docs/design/cincinnati.md

```diff
@@ -216,31 +216,31 @@ Determining valid graph traversals can be tricky to do by hand. The following [`
 
 #### Example ####
 
-The following will fetch the graph from a remote endpoint and determine the set of versions to which "1.0.0" can transition:
+The following will fetch the graph from the remote endpoint and determine the set of versions to which OpenShift version "4.2.0" can transition:
 
-```
-$ curl --silent --header 'Accept:application/json' https://cincinnati.example.com/graph | jq '. as $graph | $graph.nodes | map(.version == "1.0.0") | index(true) as $orig | $graph.edges | map(select(.[0] == $orig)[1]) | map($graph.nodes[.])'
+```json
+
+$ curl --silent --header 'Accept:application/json' https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.2 | jq '. as $graph | $graph.nodes | map(.version == "4.2.0") | index(true) as $orig | $graph.edges | map(select(.[0] == $orig)[1]) | map($graph.nodes[.])'
```

Comment:
Can you add `arch=amd64` to the parameters?

Reply:
will do

Reply:
Once I add the `arch=amd64` to the query it is not returning the pretty output `$ curl --silent --header 'Accept:application/json' https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.2&arch=amd64 | jq .`

Reply:
> `$ curl --silent --header 'Accept:application/json' https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.2&arch=amd64 | jq .`

`&` is a shell-sensitive character, so you want quotes around the URI:

```console
 $ curl --silent --header 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?arch=amd64&channel=stable-4.2' | jq ...
```

Reply:
quoted and has an explicit arch in 064b680.